### ISSUE TITLE
Provide a shim for JSDependenciesPlugin in 0.6.x

### ISSUE THREAD

**Issue by sjrd** [scala-js/scala-js#3102]
As we can see [in the instructions for cross-building between 0.6.18 and 1.0.0-M1](https://www.scala-js.org/news/2017/07/03/announcing-scalajs-1.0.0-M1/#cross-building-for-scalajs-06x-and-1x), the most complicated issue arises with the `sbt-jsdependencies` plugin, and its `JSDependenciesPlugin` auto-plugin. That's because there is no source-compatible API between sbt-scalajs 0.6.x and sbt-jsdependencies 1.0.0-M1.

To fix this, we can simply introduce an empty shim for `JSDependenciesPlugin` in sbt-scalajs 0.6.x, like this:
```scala
package org.scalajs.jsdependencies.sbtplugin

object JSDependenciesPlugin extends AutoPlugin {
  override def requires: Plugins = ScalaJSPlugin

  object autoImport
}
```
This will allow people to write `.enablePlugins(JSDependenciesPlugin)` in their `build.sbt`, and have it compile both for sbt-scalajs 0.6.x and sbt-jsdependencies 1.x.

**Comment by cquiroz** [comment]
That would be very useful

**Comment by sjrd** [comment]
Fixed in d1ebc349c3e8a59a10273761f28026fa4efccb3a.
